cmd/state/main.go

```diff
@@ -242,9 +242,9 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 
 		if childCmd.Name() != "update" && pj != nil && pj.IsLocked() {
 			if (pj.Version() != "" && pj.Version() != constants.Version) ||
-				(pj.VersionBranch() != "" && pj.VersionBranch() != constants.BranchName) {
+				(pj.VersionBranch() != "" && pj.VersionBranch() != constants.ChannelName) {
```

Comment:
Should the project methods here also be renamed?

Reply:
No. You got caught by the purpose of this ticket: to eliminate confusion between channels (state tool version branches) and branches (Platform things). The project methods refer to Platform project branches, so they should not be changed.

Reply:
I was going to say no, but you're right. I thought all projectfile branch methods were related to Platform project branches, but apparently there's a way to lock projectfiles to State Tool channels. I learned something new.